LongWarOfTheChosen/Src/LW_PerkPack_Integrated/Classes/X2Effect_GreaterPadding.uc

```diff
@@ -0,0 +1,124 @@
+///---------------------------------------------------------------------------------------
+//  FILE:    X2Effect_GreaterPadding
+//  AUTHOR:  Grobobobo (taken from field surgeon)
+//  PURPOSE: Implements effect for FieldSurgeon ability -- this allows healing of lowest hp at end of mission
```

Comment:
This should explain what the effect does, particularly how it differs from Field Surgeon.